net/ip_utils.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+const IPV4_REGEX = /^[0-9]{0,3}\.[0-9]{0,3}\.[0-9]{0,3}\.[0-9]{0,3}$/;
```

Comment:
How did you get the regex? Genuine question. Do you have a reference that you can link to?